tck/src/test/resources/suites/tck-base-suite.xml

```diff
@@ -19,16 +19,22 @@
     <test name="base-tests" verbose="10">
         <groups>
             <define name="base-groups">
+                <include name="utils" description="Utility tests" />
                 <include name="ejb" description="EJB container integration tests" />
             </define>
             <define name="excludes">
+                <include name="utils-extra" description="Additional utility tests" />
             </define>
             <run>
                 <include name="base-groups" />
                 <exclude name="excludes" />
             </run>
         </groups>
         <classes>
+            <class name="org.eclipse.microprofile.jwt.bridge.tck.util.TokenUtilsTest" />
```

Comment:
Should we really make these utility tests part of the TCKs that the implementations will have to pass ? My understanding these utility tests will only run during the TCK builds ?

Reply:
@ayoho Never mind, we do it in MP JWT too, kind of makes sense to run them too to make sure the token generation is sound before the actual TCK runs